plasmapy/particles/decorators.py

```diff
@@ -351,7 +367,7 @@ def verify_charge_categorization(self, particle) -> None:
 
         if isinstance(uncharged, Iterable):
             uncharged = any(uncharged)
-            lacks_charge_info = any(lacks_charge_info)
+            lacks_charge_info = any(lacks_charge_info)  # type: ignore[arg-type]
```

Comment:
For some reason, the built-in `any` and `all` annotations aren't matching the types that we're using here.

Reply:
🤔